src/osd/osd_types.h

```diff
@@ -3835,6 +3835,14 @@ class pg_missing_set : public pg_missing_const_i {
       return false;
     return true;
   }
+  eversion_t get_oldest_need() {
```

Comment:
@xiexingguo could you make this method `const`?

Reply:
@tchaikov Sure, done.